docs/concept/wal.md

```diff
@@ -0,0 +1,56 @@
+# Wal Format
+
+![](/assets/wal-format.jpg)
+
+Wal file consist of a series of WalEntry，WalEntry contain as follows:
+
+- Length:8 bytes,means length of a WalEntry.
+- Series ID:8 bytes,the same as request Series ID.
+- Count:4 bytes,how many binary/timestamp in one WalEntry. 
+- Timestamp:8 bytes.
+- Binary Length:2 bytes.
+- Binary:value in write request.
+
+# Write process
+
+![](/assets/wal.jpg)
+
+When client send a request,the the request will be sent to the tsdb instance corresponding to the group.And when tsdb receives a write request, the following steps occur:
```

Comment:
It's not good practice to mention the tsdb and group. Regularly, WAL should not presume any dedicated users or invokers.

Reply:
This section should focus on the internal component instead of the external ones.